src/test/kotlin/com/karumi/maxibonkata/DevelopersTest.kt

```diff
@@ -0,0 +1,37 @@
+package com.karumi.maxibonkata
+
+import io.kotlintest.matchers.shouldBe
+import io.kotlintest.properties.forAll
+import io.kotlintest.specs.ShouldSpec
+
+class DevelopersTest : ShouldSpec() {
+    init {
+
+        "Developers" {
+            should("always grab a positive number of maxibons") {
+                forAll({ numberOfMaxibons: Int ->
+                    val dev = Developer(numberOfMaxibons = numberOfMaxibons)
+
+                    dev.maxibonsToGrab >= 0
+                })
+            }
+
+            should("assign the name of the developer in construction") {
+                forAll({ name: String ->
+                    val dev = Developer(name)
+
+                    dev.name == name
+                })
+            }
+
+            should("assign the number of maxibons specified to every developer") {
```

Comment:
you could use [TableDriven](https://github.com/kotlintest/kotlintest/blob/master/doc/reference.md#table-driven-testing-) here to define every test case. You'd simplify the test and get a different test execution for every case. You'd also avoid masking errors.

Reply:
I like the idea 😄